GSAGH-454-Change-unit-of-Moment-per-Length-to-kNm-m-etc

tlmnrnhrdt opened the pull request.

tlmnrnhrdt commented:
@tim-lyon @RaviKumarGubbala Hi both, I added MomentPerLength to OasysUnits since the lack of this unit was confusing our using and myself regularly. The magic numbers for the conversion to imperial units need thorough checking!

DominikaLos commented on OasysUnits.NanoFramework/GeneratedCode/AreaDensity/UnitsNet.NanoFramework.AreaDensity.nuspec:
```diff
@@ -2,7 +2,7 @@
   <metadata>
     <id>OasysUnits.nanoFramework.AreaDensity</id>
-    <version>5.12.0</version>
+    <version>1.1.1</version>
```
why from 5.12 -> 1.1? 
the same for other places

tlmnrnhrdt replied:
@DominikaLos 5.12.0 is the UnitsNet version, it should have been 1.1.0 instead of 5.12.0 before..

tlmnrnhrdt commented:
> @tim-lyon @RaviKumarGubbala Hi both, I added MomentPerLength to OasysUnits since the lack of this unit was confusing our users and myself regularly. The magic numbers for the conversion to imperial units need thorough checking!

I added additional tests and derived those "magic numbers" from existing constants of other tests.

angelahu005 commented on Common/UnitDefinitions/MomentPerLength.json:
```diff
@@ -0,0 +1,281 @@
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "kipf·ft/ft", "kip·ft/ft", "k·ft/ft" ]
```
most commonly kip.ft/ft
